test/oc_stat_test_exporter.erl

```diff
@@ -0,0 +1,6 @@
+-module(oc_stat_test_exporter).
```

Comment:
A nitpick, but since we have `oc_pid_reporter` and `oc_tab_reporter` that does the same logic it would be clearer to have this be `oc_stat_pid_exporter.erl`.